### PR TITLE
Fix NullPointer /gw console

### LINKED ISSUE #31: NullPointerException when running /gw from the console
When running one of the plugin commands from the console, a NullPointerException occurs.

**To Reproduce**
1. Run /gw in the console
2. Command fails with an NPE

**Expected behavior**
Instead of throwing an exception, a message would be sent saying that a player needs to execute the command.

Exception:
```
[12:35:35 ERROR]: [GlobalWarming] [ACF] Exception in command: gw 
[12:35:35 ERROR]: [GlobalWarming] [ACF] java.lang.NullPointerException
[12:35:35 ERROR]: [GlobalWarming] [ACF] 	at net.porillo.GlobalWarming.lambda$registerCommands$1(GlobalWarming.java:122)
[12:35:35 ERROR]: [GlobalWarming] [ACF] 	at net.porillo.shade.aikar.commands.RegisteredCommand.resolveContexts(RegisteredCommand.java:296)
[12:35:35 ERROR]: [GlobalWarming] [ACF] 	at net.porillo.shade.aikar.commands.RegisteredCommand.resolveContexts(RegisteredCommand.java:215)
[12:35:35 ERROR]: [GlobalWarming] [ACF] 	at net.porillo.shade.aikar.commands.RegisteredCommand.invoke(RegisteredCommand.java:149)
[12:35:35 ERROR]: [GlobalWarming] [ACF] 	at net.porillo.shade.aikar.commands.BaseCommand.executeCommand(BaseCommand.java:576)
[12:35:35 ERROR]: [GlobalWarming] [ACF] 	at net.porillo.shade.aikar.commands.BaseCommand.execute(BaseCommand.java:512)
[12:35:35 ERROR]: [GlobalWarming] [ACF] 	at net.porillo.shade.aikar.commands.RootCommand.execute(RootCommand.java:99)
[12:35:35 ERROR]: [GlobalWarming] [ACF] 	at net.porillo.shade.aikar.commands.BukkitRootCommand.execute(BukkitRootCommand.java:79)
[12:35:35 ERROR]: [GlobalWarming] [ACF] 	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:151)
[12:35:35 ERROR]: [GlobalWarming] [ACF] 	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:734)
[12:35:35 ERROR]: [GlobalWarming] [ACF] 	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchServerCommand(CraftServer.java:696)
[12:35:35 ERROR]: [GlobalWarming] [ACF] 	at net.minecraft.server.v1_13_R2.DedicatedServer.handleCommandQueue(DedicatedServer.java:483)
[12:35:35 ERROR]: [GlobalWarming] [ACF] 	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:440)
[12:35:35 ERROR]: [GlobalWarming] [ACF] 	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:940)
[12:35:35 ERROR]: [GlobalWarming] [ACF] 	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:837)
[12:35:35 ERROR]: [GlobalWarming] [ACF] 	at java.lang.Thread.run(Thread.java:748)
```

### PR DESCRIPTION
**Pull request by Dariasc**
Fixes #31 
Todo: Replace with CommandSender where needed

### CODE REVIEW

File: src/main/resources/lang.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ lang:
     INVALIDARGS: '§c§OInvalid command arguments, expecting: §7§O%s'
     SPAM: §8§OToo many requests at once
     INVENTORYFULL: §c§OYour inventory is full
+    PLAYERONLY: Player only command
```
Comment (nsporillo) [inline src/main/resources/lang.yml]: Should we have a more detailed and formatted error message here?

File: src/main/java/net/porillo/GlobalWarming.java [hunk 2]
```diff
@@ -133,7 +137,13 @@ public void onDisable() {
     private void registerCommands() {
         commandManager.enableUnstableAPI("help");
         commandManager.setFormat(MessageType.HELP, ChatColor.GREEN, ChatColor.DARK_GREEN, ChatColor.AQUA);
-        commandManager.getCommandContexts().registerIssuerOnlyContext(GPlayer.class, c -> tableManager.getPlayerTable().getPlayers().get(c.getPlayer().getUniqueId()));
+        commandManager.getCommandContexts().registerIssuerOnlyContext(GPlayer.class, c -> {
+            CommandSender sender = c.getSender();
+            if (sender instanceof Player) {
+                return tableManager.getPlayerTable().getPlayers().get(c.getPlayer().getUniqueId());
```
Comment (nsporillo) [inline src/main/java/net/porillo/GlobalWarming.java]: This is a nice solution, perhaps we could create a generic GPlayer to be used for any CommandSender?

Would just need to be careful for when we actually use the GPlayer in the command classes, maybe just add an additional check for all the commands which strictly need an ingame player.

File: src/main/resources/lang.yml (hunk 1 above)
Comment (Dariasc) [reply src/main/resources/lang.yml]: ACF seems to add some formatting to the exception thrown in the Context, it ends up being a red Error: Player only command, we could definitely change the text, idk what to though

File: src/main/java/net/porillo/GlobalWarming.java (hunk 2 above)
Comment (Dariasc) [reply src/main/java/net/porillo/GlobalWarming.java]: We can replace GPlayer with CommandSender wherever we want everybody to execute the command and ACF will resolve it for us

Comment (Dariasc) [reply src/main/java/net/porillo/GlobalWarming.java]: We might have to change isCommandAllowed method, we could replace that method with a custom ACF flag, making it cleaner and simpler

**Pull request merged**